qa/tasks/cephfs/test_volumes.py

```diff
@@ -63,7 +63,29 @@ def  _get_subvolume_path(self, vol_name, subvol_name, group_name=None):
         return path[1:].rstrip()
 
     def _delete_test_volume(self):
-        self._fs_cmd("volume", "rm", self.volname)
+        try:
+            self._fs_cmd("volume", "rm", self.volname)
```

Comment:
The reason you're not seeing this in the teuthology logs is there is no test that deletes a volume. You need to write one.

Also, we don't really want to do this test for `--yes-i-really-mean-it` every single time we need to delete a volume. Let's just write a specific test that confirms.

Reply:
I will write a test function `test_` that deletes a volume. But I think `_delete_test_volume` should be modified this way, as `fs volume rm` works only with `--yes-i-really-mean-it` now onwards?

Reply:
Correct but it should just be this change:

````
-self._fs_cmd("volume", "rm", self.volname)
+self._fs_cmd("volume", "rm", self.volname, "--yes-i-really-mean-it")
````

Reply:
@batrick Fixed. The tests are in progress now.